src/pino_transport.ts

```diff
@@ -0,0 +1,100 @@
+import build from "pino-abstract-transport"
+import { Client } from "./client"
+
+type PinoTransportOptions = {
+  client: Client
+  group: string
+}
+
+const appsignalPinoTransport = ({
+  client,
+  group = "app"
+}: PinoTransportOptions) => {
+  let buffer: Record<string, any>[] = []
+  let timeout: NodeJS.Timeout | null = null
+
+  return build(
+    async (source: any) => {
+      for await (const obj of source) {
+        buffer.push(parseInfo(obj, group))
```

Comment:
Do we need to buffer the lines before sending them? I see that https://github.com/duailibe/pino-appsignal-transport does it, because it's sending an HTTP request.

But we're sending them to the extension, where the agent will already do its own buffering before sending them over the network. I think we could simplify the implementation here by removing the buffering logic and sending them to the agent straight away.

Reply:
I'll try, and if it works, we'll keep it. The simpler the better.